test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,81 @@
+const agent = require('superagent');
+const chai = require('chai');
+const md5 = require('md5');
+
+chai.use(require('chai-subset'));
+
+const { expect } = chai;
+
+const url = 'https://api.github.com';
+
+describe('Given a user the github', () => {
+  const username = 'aperdomob';
+
+  describe('when get user', () => {
+    let repositories;
+    let repository;
+    let user;
+    const repositorySearch = 'jasmine-awesome-report';
+
+    before(async () => {
+      const userResponse = await agent.get(`${url}/users/${username}`).auth('token', process.env.ACCESS_TOKEN).set('User-Agent', 'agent');
+      user = userResponse.body;
+      const repositoriesResponse = await agent.get(`${url}/users/${username}/repos`).auth('token', process.env.ACCESS_TOKEN).set('User-Agent', 'agent');
```

Comment:
This is a different task and you are veryfying other thinks, so I would suggest to make the part of the repositories on another describe after the validations of the user

Reply:
Also, try to get the url by hypermedia (using the response you got from the user)